Packs/FeedCSV/Integrations/FeedCSV/FeedCSV.yml

```diff
@@ -129,19 +129,25 @@ configuration:
   name: skipinitialspace
   type: 8
   required: false
-- additionalinfo: Supports CSV values.
-  display: Tags
+- additionalinfo: Text encoding for the CSV file. Defaults to latin-1.
+  display: Text Encoding
+  name: encoding
+  type: 0
+  required: true
```

Comment:
shouldn't be required 
```suggestion
```

Reply:
If it is not required, it's possible to create an instance with an empty field which always fails. If you want to remove required, we'd need to add extra logic to the code to treat an empty string as latin-1.

Reply:
Ok
as the param have default value (which mean that the existing users will not need to touch the configuration)
you are right